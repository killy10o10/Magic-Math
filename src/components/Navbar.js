import React from 'react';
import { Link } from 'react-router-dom';

const Navbar = () => (
  <header className="header">
    <nav id="logo">
      <Link to="/">Math Magicians</Link>
    </nav>
    <nav className="nav-links">
      <Link className="nav" to="/">
        Home
      </Link>
      <Link className="nav" to="/calculator">
        Calculator
      </Link>
      <Link className="nav" to="/quote">
        Quote
      </Link>
    </nav>
  </header>
);

export default Navbar;
