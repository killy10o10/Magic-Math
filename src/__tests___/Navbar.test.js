import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('Testing the Navbar', () => {
  it('Nav is rendered', () => {
    const nav = renderer
      .create(
        <BrowserRouter>
          <Navbar />
        </BrowserRouter>,
      )
      .toJSON();
    expect(nav).toMatchSnapshot();
  });
});
