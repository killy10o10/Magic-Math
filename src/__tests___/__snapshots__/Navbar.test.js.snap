// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Navbar Nav is rendered 1`] = `
<header
  className="header"
>
  <nav
    id="logo"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Math Magicians
    </a>
  </nav>
  <nav
    className="nav-links"
  >
    <a
      className="nav"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="nav"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="nav"
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</header>
`;
